_data-prepper/pipelines/configuration/sources/s3.md

```diff
@@ -137,18 +146,59 @@ These options map directly to options available in the S3 Select [CSVInput](http
-quote_escape | No  | String  | A single character used for escaping the quotation mark character inside an already escaped value. Maps directly to the [QuoteEscapeCharacter](https://docs.aws.amazon.com/AmazonS3/latest/API/API_CSVInput.html#AmazonS3-Type-CSVInput-QuoteEscapeCharacter) property.
-comments | No   | String  | A single character used to indicate that a row should be ignored when the character is present at the start of that row. Maps directly to the [Comments](https://docs.aws.amazon.com/AmazonS3/latest/API/API_CSVInput.html#AmazonS3-Type-CSVInput-Comments) property.
+`file_header_info` | No  | String  | Describes the first line of input. Maps directly to the [FileHeaderInfo](https://docs.aws.amazon.com/AmazonS3/latest/API/API_CSVInput.html#AmazonS3-Type-CSVInput-FileHeaderInfo) property.
+`quote_escape` | No  | String  | A single character used for escaping the quotation mark character inside an already escaped value. Maps directly to the [QuoteEscapeCharacter](https://docs.aws.amazon.com/AmazonS3/latest/API/API_CSVInput.html#AmazonS3-Type-CSVInput-QuoteEscapeCharacter) property.
+`comments` | No   | String  | A single character used to indicate that a row should be ignored when the character is present at the start of that row. Maps directly to the [Comments](https://docs.aws.amazon.com/AmazonS3/latest/API/API_CSVInput.html#AmazonS3-Type-CSVInput-Comments) property.
 
 #### json<a name="s3_select_json"></a>
 
 Use the following option in conjunction with `json` for `s3_select` to determine how S3 Select processes the JSON file.
 
-Option | Required | Type  | Description
-:--- |:---------|:------------| :---
-type | No  | String  | The type of JSON array. May be either `DOCUMENT` or `LINES`. Maps directly to the [Type](https://docs.aws.amazon.com/AmazonS3/latest/API/API_JSONInput.html#AmazonS3-Type-JSONInput-Type) property.
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+`type` | No  | String  | The type of JSON array. May be either `DOCUMENT` or `LINES`. Maps directly to the [Type](https://docs.aws.amazon.com/AmazonS3/latest/API/API_JSONInput.html#AmazonS3-Type-JSONInput-Type) property.
+
+## Using `scan` with the `s3` source<a name="scan"></a>
+The following parameters allow you to scan S3 objects. All options ban be configured at the bucket level.
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+`start_time` | No  | String | The start of the last modified time to scan objects from all the buckets. This should follow [ISO LocalDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME) format, for example, `023-01-23T10:00:00`, or it can be configured to the keyword which represents current `LocalDateTime`, such as `now`. To define a time range, use `start_time` with  either `end_time` or `range`. 
```

Comment:
```suggestion
`start_time` | No  | String | The start of the last modified time to scan objects from all the buckets. This should follow [ISO LocalDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME) format, for example, `023-01-23T10:00:00`, or it can be configured to the keyword that represents the current `LocalDateTime`, such as `now`. To define a time range, use `start_time` with either `end_time` or `range`. 
```